core/store/migrations/migration1592355365/migrate.go

```diff
@@ -0,0 +1,22 @@
+package migration1592355365
+
+import (
+	"github.com/jinzhu/gorm"
+)
+
+// Migrate ensures that heads are unique and adds parent hash for use in reorg detection
+func Migrate(tx *gorm.DB) error {
+	return tx.Exec(`
+		CREATE TABLE "job_spec_errors" (
+			"id" uuid primary key NOT NULL,
+			"job_spec_id" uuid REFERENCES job_specs(id) ON DELETE CASCADE NOT NULL,
+			"description" text NOT NULL,
+			"occurances" integer DEFAULT 1 NOT NULL,
+			"created_at" timestamp without time zone NOT NULL,
```

Comment:
Use `timestamptz` data type (shorthand for `timestamp with time zone`).

These names are incredibly misleading, but `timestamp with time zone` is almost always the correct one to use.